imgdoc2API/imgdoc2API.h

```diff
@@ -63,6 +65,19 @@ typedef bool(LIBIMGDOC2_STDCALL* MemTransferReserveFunctionPointer)(std::intptr_
 /// Defines an alias representing a function pointer used for memory transfer operations. This function pointer is used with IDocRead2d_ReadTileData/IDocRead3d_ReadBrickData.
 typedef bool(LIBIMGDOC2_STDCALL* MemTransferSetDataFunctionPointer)(std::intptr_t /*blob_output_handle*/, std::uint64_t /*offset*/, std::uint64_t /*size*/, const void* /*data*/); // NOLINT(readability/casting)
 
+/// Defines an alias representing a function pointer used for memory allocation (on the caller side).
+typedef bool(LIBIMGDOC2_STDCALL* AllocMemoryFunctionPointer)(std::uint64_t size, AllocationObject* allocation_object); // NOLINT(readability/casting)
+
+/// Gets version information. This function is used to retrieve the version information of the library.
+/// Note that the specified allocate_memory_function is used to allocate memory for the strings in the VersionInfoInterop structure.
+/// If the allocate_memory_function is null, then the strings will not be returned.
+///
+/// \param [out]    version_info                Structure where the version information will be placed.
+/// \param          allocate_memory_function    If non-null, this "allocate memory function" will be used to allocate memory for the strings in the VersionInfoInterop structure
+///  
+/// \returns An error-code indicating success or failure of the operation.                                                                                       to be returned.
```

Comment:
Unclear of the purpose of the to be returned on the far right here.

Reply:
No purpose I can think of. Fixed, thanks.